packages/lesswrong/components/comments/RecentDiscussionThread.jsx

```diff
@@ -177,7 +170,6 @@ class RecentDiscussionThread extends PureComponent {
                   children={comment.children}
                   key={comment.item._id}
                   updateComment={updateComment}
-                  refetch={refetch}
```

Comment:
I'm not sure if refetch still naively works here, but note that the current version of this PR breaks the functionality that it was doing (i.e. if you reply to a comment in recent discussion, the comment now disappears until you refresh the page, which is what the refetch was supposed do address)